example/src/main/java/io/netty/example/http2/helloworld/server/Http2Server.java

```diff
@@ -44,7 +44,7 @@
 
     static final boolean SSL = System.getProperty("ssl") != null;
 
-    static final int PORT = Integer.parseInt(System.getProperty("port", SSL? "8443" : "8080"));
+    public static final int PORT = Integer.parseInt(System.getProperty("port", SSL? "8443" : "8080"));
```

Comment:
Could you explain why you needed this? I recommend adding a new static final property for each different example.